ament_tools/build_types/cmake.py

```diff
@@ -91,13 +97,19 @@ def extend_context(self, options):
         ce.add('force_cmake_configure', force_cmake_configure)
         ce.add('cmake_args', options.cmake_args)
         ce.add('ctest_args', options.ctest_args)
+        ninja_build = False
+        if getattr(options, 'ninja', False):
+            ninja_build = True
+        ce.add('ninja_build', ninja_build)
```

Comment:
Please call the context variable also `use_ninja` - same as the updated command line argument.